tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -3004,4 +3011,13 @@ public static function isExpectedToFail(string $scenarioLine): bool {
 		}
 		return false;
 	}
+
+	/**
+	 * @return string
+	 */
+	public function getExpiryDateTime(): string {
+		$expiryDateTime = new DateTime('yesterday');
+		//        $expiryDateTime->setTime(5, 45, 0);
+		return $expiryDateTime->format('Y-m-d H:i:s');
```

Comment:
this method should only return already defined date
```suggestion
		return $this->expiryDateTime;
```